php/pantheon/checks/objectcache.php

```diff
@@ -30,10 +30,10 @@ public function run() {
 			$this->alerts[] = array("message"=> "object-cache.php exists", "code" => 0);
 		}
 
-		if ( ! defined( 'WP_REDIS_OBJECT_CACHE' ) || ! WP_REDIS_OBJECT_CACHE ) {
-			$this->alerts[] = array("message"=> 'Use Redis with the WP Redis object cache drop-in to speed up your backend. <a href="https://pantheon.io/docs/wordpress-redis/" target="_blank">Learn More</a>', "code" => 1);
+		if ( ! defined( 'WP_REDIS_CONFIG' ) || ! WP_REDIS_CONFIG ) {
```

Comment:
I think I might keep a check for WP Redis in, but change the message to something like "WP Redis found. We recommend using Object Cache Pro instead..." e.g.:

```suggestion
		} else {
				$this->alerts[] = array('message'=> 'WP Redis for object caching was found. We recommend using Object Cache Pro as a replacement. <a href="https://docs.pantheon.io/guides/object-cache-pro/installing-configuring/" target="_blank">Learn More</a> about how to install Object Cache Pro.', 'code' => 1);
		}
		if ( ! defined( 'WP_REDIS_CONFIG' ) || ! WP_REDIS_CONFIG ) {
```